index.js

```diff
@@ -69,6 +68,13 @@ async function openPopup() {
 		notificationsList = select.all('.boxed-group', notificationsPage);
 		if (notificationsList.length === 0) {
 			return;
+		} else if (!select('.notification-indicator .mail-status').classList.contains('unread')) {
+			const newDom = await fetch(document.URL, {
+				credentials: 'include'
+			}).then(r => r.text()).then(domify);
+
+			indicator.parentNode.replaceChild(select('.notification-indicator', newDom), indicator);
+			init();
```

Comment:
Why is this needed?

Reply:
I think when I replace the `.notification-indicator`, the old event listeners are gone (at least that's what I interpreted when it didn't work the next time). Hence, before I hover on the icon next time, it should register the events to keep it funcitoning..